src/platform/forms-system/src/js/widgets/RadioWidget.jsx

```diff
@@ -99,7 +99,7 @@ RadioWidget.propTypes = {
     widgetProps: PropTypes.shape({}),
     selectedProps: PropTypes.shape({}),
     enableAnalytics: PropTypes.bool,
-    title: PropTypes.string,
+    title: PropTypes.string || PropTypes.element,
```

Comment:
Gets rid of the false-positive console-error for invalid prop-type when providing JSX for the title.